pkg/cmd/build/logger.go

```diff
@@ -130,6 +130,11 @@ func (t *trace) display() {
 			}
 		}
 		if t.hasCommandLogs(v) {
+			oktetoLog.StopSpinner()
+			logs := strings.Join(v.logs, "")
+			if !strings.HasPrefix(logs, "{") {
```

Comment:
Maybe we should add `--log-level` in the destroy command. That way it'd be already contemplated in lines 140-160